integrations/pull_merge_test.go

```diff
@@ -58,7 +59,34 @@ func TestPullMerge(t *testing.T) {
 
 	elem := strings.Split(test.RedirectURL(resp), "/")
 	assert.EqualValues(t, "pulls", elem[3])
-	testPullMerge(t, session, elem[1], elem[2], elem[4])
+	testPullMerge(t, session, elem[1], elem[2], elem[4], "merge")
```

Comment:
If it's not too hard, could we add check the result of merge (that a commit was created, that correct merge style was used, etc.)?

Reply:
@lafriks Is this doable?